packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -443,12 +588,15 @@ export function getNotSeenValidatorsFn(state: CachedBeaconStateAllForks): GetNot
       state
     );
 
-    return (epoch: Epoch, committee: Uint32Array) => {
+    return (epoch: Epoch, slot: Slot, committeeIndex: number) => {
       const participants =
         epoch === stateEpoch ? currentEpochParticipants : epoch === stateEpoch - 1 ? previousEpochParticipants : null;
       if (participants === null) {
         return null;
       }
+      const shuffling = state.epochCtx.getShufflingAtEpoch(epoch);
+      const slotCommittees = shuffling.committees[slot % SLOTS_PER_EPOCH];
+      const committee = slotCommittees[committeeIndex];
```

Comment:
I think these 3 lines can be replaced by ` const committee = epochCtx.getBeaconCommittee(slot, committeeIndex)` here. Same to L638-L642

Reply:
good point 👍